RealmSwift/Tests/ObjectiveCSupportTests.swift

```diff
@@ -0,0 +1,243 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import XCTest
+import Realm
+@testable import RealmSwift
```

Comment:
The `RealmSwift` module isn't compiled with testability enabled. You'll either need to update the `xcconfig` files to compile with `ENABLE_TESTABILITY` enabled, or to refactor these tests to avoid needing to access `internal` members.

We've avoided enabling testability so far to prevent accidentally shipping compiled frameworks with extra debug information, or lower optimizations, so those concerns would need to be addressed if you end up wanting to go that route.